Interview/ConductingATechnicalInterview.md

```diff
@@ -0,0 +1,24 @@
+Outline:
```

Comment:
Probably not needed(?)

Reply:
Just a market that it is an outline and not a draft document.